src/main/java/com/maveric/dto/ForgotPassword.java

```diff
@@ -0,0 +1,19 @@
+package com.maveric.dto;
+
+import jakarta.validation.constraints.NotBlank;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+
+public class ForgotPassword {
+    @NotBlank(message = "Email is required")
+    private String email;
+  //  @NotBlank(message = "Password is required")
```

Comment:
Remove ghost code here

Reply:
newPasswordOne and newpasswordtwo feels bit ambiguous can you come up with good naming convention.